robot/Crisis-Management/tests/browser/t3_availability_records_created.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+
+Resource        robot/Crisis-Management/resources/CrisisManagement.robot
+Library
+...             cumulusci.robotframework.PageObjects
+...             robot/Crisis-Management/resources/ContactPageObject.py
+...             robot/Crisis-Management/resources/CrisisManagement.py
+
+Suite Setup     Run keywords
+...             Open Test Browser
+
+Suite Teardown  Delete Records and Close Browser
+
+*** Variables ***
+${Last_name}   tractionuser
```

Comment:
now u can also use faker to generate realistic fake data. Just a suggestion

Reply:
Updates to choose faker